test/e2e/scenarios/cmd_basic_create_update.yaml

```diff
@@ -0,0 +1,40 @@
+id: "RG_CMD_basic_create_update"
+description: "In this test we create RG with absolutely minimum configuration and try to update it."
+usecases:
+  - replication_group
+  - cmd
+#marks:
+#  - slow
+#  - blocked
+customResourceReference:
+  apiVersion: $CRD_GROUP/$CRD_VERSION
+  kind: ReplicationGroup
+  metadata:
+    name: test$RANDOM_SUFFIX
+steps:
+  - id: "create_CMD_replication_group"
+    description: "Initial config"
+    create:
+      spec:
```

Comment:
+1 to Jay's comment about this being in a separate resource file.

Reply:
Added support for resource yaml file name as value.